test/mlir/test/gc/Dialect/CPURuntime/memref-to-cpuruntime.mlir

```diff
@@ -0,0 +1,23 @@
+// RUN: gc-opt --split-input-file --convert-memref-to-cpuruntime %s -verify-diagnostics | FileCheck %s
+
+func.func @alloc() {
+  // CHECK-LABEL: func @alloc()
+  // CHECK: %[[m0:.*]] = cpuruntime.alloc() : memref<1024xf32>
+  %m0 = memref.alloc() : memref<1024xf32>
```

Comment:
Please also check the dynamic shaped case?

Reply:
added.

Reply:
@crazydemo, could you please point me to the dynamic shaped case?